docs/reference/requirement-sets/add-in-commands-requirement-sets.md

```diff
@@ -28,6 +28,17 @@ The following table lists the add-in commands 1.1 requirement set, the Office cl
 
 >\* The [Office.context.requirements.isSetSupported](/javascript/api/office/office.requirementsetsupport#issetsupported-name--minversion-) method will erroneously return `false` for versions 16.9 &ndash; 16.14 (inclusive), but the requirement set *is* supported on these versions.
 
+The add-in commands 1.3 requirement set introduces manifest markup that enables an add-in to customize the placement of a custom tab on the Office ribbon and to insert built-in Office ribbon controls into custom control groups.
+
+The following table lists the add-in commands 1.3 requirement set, the Office client applications that support that requirement set, and the build or version numbers for the Office application.
+
+|  Requirement set  |  Office 2013 and later on Windows<br>(one-time purchase) | Office on Windows<br>(connected to a Microsoft 365 subscription)   |  Office on iPad<br>(connected to a Microsoft 365 subscription)  |  Office on Mac<br>(connected to a Microsoft 365 subscription)  | Office on the web  |  
```

Comment:
Is there a reason why the info about 1.1 and 1.3 (if not all of them, including the initial release) can't be included in the same table?

Reply:
I thought I remembered being told that, for another req set, that they had to be separate tables, but now I can't find it and it looks like we have a single table for other req sets, so I'll combine them.